### ISSUE TITLE
Add logging to measure runtime; add min_epochs param via config file; format metric values to 3 significant digits

### ISSUE THREAD

**Issue by malabikasen** [Murali-group/supervised-grns#17]
1. Create a logging file to note down run time values of training.
2. Create a parameter in the config file called `min_epochs` which is the minimum number of epochs to be trained until the validation loss stabilizes.
3. Modify the metric values to include 3 digits of precision.
